### ISSUE TITLE
missed complete themes "addons"

### ISSUE THREAD

**Issue by pirate486743186** [JustOff/ca-archive#28]
complete themes
https://addons.mozilla.org/en-US/firefox/complete-themes/

they used to be "complete themes" as opposed to light themes. In 2015 they where dropped.... kinda. They where converted to normal addons.

CAA completely missed them.... Mozilla too seams to have missed them when they deleted all the xul addons.... You better scrape them before they notice XD.... They are 493

**Comment by JustOff** [comment]
Sorry, but themes are beyond the scope of this project.

**Comment by JustOff** [comment]
I am aware of what the full themes are, and I have reasons not to include them into CAA.

**Comment by Vangelis66** [comment]
> not to include them into CAA

... Of course, I'm not the one _to question the owner's decisions_ :wink: ; let me just say how grateful I am for this project, which proved to me its immense value again today :+1: : at the time of writing (Nov 29th, 2018), **ALL non-Quantum compatible** extensions are _not to be found on AMO_ anymore 😭 ; this does not only include XUL and jetpack ones, but even WebExtension ones that wouild install in Fx 48.0-56.0 and in **Basilisk**/UXP... 
As expected, (**complete**) **themes were not spared**, so this is how I arrived in this issue's comments section... 

From using this extension (v**1.2.2**) today, I noticed the catalogued legacy extensions are served by a `ca-archive.biz.tm` hostname, which itself redirects to a `storage-waterfox.netdna-ssl.com` hostname, making it safe to assume it points to an AMO back-up put in place by the Waterfox dev(s) (:+1:) ... I am not familiar with the Waterfox project, being on a 32-bit OS, but I want to kindly ask: 
1. Given that the Waterfox browser seems capable of installing third party complete themes, has it come to your knowledge whether the Waterfox AMO archiving project has actually backed up Complete Themes, too? 
2. If _yes_ to (1.), is there some means to access those complete themes now (obviously outside this extension, since you are not inclined to support them - a decision I totally respect 😃 )? 

I had always used 3rd party (mostly **dark**, due to vision issues)  themes in my Firefox/Pale Moon/**Basilisk** profiles, I, of course, maintain local back-ups of said themes, but accidents _do_ happen (e.g. HDD failures etc.), so having an online source to resort to would be comforting when the worst happens... 

Most thankful for any eventual answer to my "Complete Themes" queries, may you be blessed for what you're doing for the benefit of _real_ browsers (i.e. not Chrome clones 😞 ) !

Best regards

**Comment by pirate486743186** [comment]
complete themes and web extentions, are probably all on wayback machine, This is probably true for all files that were available in 2018. When trying to download them, use the full url with the "?stuff", the crawler actually created  files like this "extension-name.xpi?stuff".

**Comment by Vangelis66** [comment]
> complete themes [and web extentions,] are probably all on wayback machine, 

... Well, of course, I already know that :smile:; FWIW, latest WBM capture for "Complete Themes" is just from **5** days ago (for the en-US locale, anyway): 

`https://web.archive.org/web/20181124130244/https://addons.mozilla.org/en-US/firefox/complete-themes/`

But, sadly, **not** _every_ version for _every_ complete theme released is accessible via that route... My query is for a more "resilient"/reliable access solution (provided, hopefully, by a specially archived repository, if such even exists...); if the internet archive is all we're left with, it's by all means better than nothing at all... :wink:

**Comment by grahamperrin** [comment]
[Legacy add-ons in the Internet Archive Wayback Machine : waterfox](https://www.reddit.com/r/waterfox/comments/adznrq/legacy_addons_in_the_internet_archive_wayback/)

> … Not all legacy add-ons are in the Wayback Machine, but installation does work for some of the add-ons that are not in the Classic Add-ons Archive (CAA). …

– and this includes some complete themes. 

tl;dr green button, open link in new tab.

**Comment by pirate486743186** [comment]
I think that all complete themes that were at mozilla a few months ago are in the wayback machine. Unless some one has found a counter example?

**Comment by grahamperrin** [comment]
> https://web.archive.org/web/20181124130244/https://addons.mozilla.org/en-US/firefox/complete-themes/

– we have that (the first of thirty-one pages) last captured _with results_ in November, then for page 2 it's a step back to mid-October 2018: 

- https://web.archive.org/web/20181015142316/https://addons.mozilla.org/en-US/firefox/complete-themes/?page=2

– jump to the last page, also captured on the 15th: 

- https://web.archive.org/web/20181015142307/https://addons.mozilla.org/en-US/firefox/complete-themes/?page=31

– from near the foot, the page for iPox captured on the 18th: 

- https://web.archive.org/web/20181018230554/https://addons.mozilla.org/en-US/firefox/addon/ipox/?src=cb-dl-users

– with its `.jar` file captured around twenty seconds later. 

----

Complete themes aside, recent https://github.com/JustOff/ca-archive/commit/28e4837bff066cf80232773f7f60fb93471e8c69 was: 

> Open the Wayback Machine links with the date closest to October 2018
